README.md

```diff
@@ -105,6 +105,49 @@ There are a couple of cases where this technique might be necessary:
 
 Note: when attaching a handler using the binding, you will need to manage the value of `this` as you would when using the normal `click` and `event` bindings.  So, in this case you would have to take care of it in our view model or bind against it like `$parent.removeItem.bind($parent)`.
 
+###4 - Specify a function reference in `delegatedFatherTheEventName` binding and use `data-TheEventName-father` attribute
+
+The final option for specifying a handler is to place a normal binding on the father element that stores a reference to the function with the element (but does not attach an event handler). A `data-TheEventName-father` attribute should be added to the element that should respond to the event. Exactly as for `delegateTheEventName`, bindings for each event will be dynamically created, as necessary, by the `delegatedHandler` binding.
+It is possible to handle mutiple child elements in a single father element using object literal as `delegatedFatherTheEventName` binding where keys should be used as `data-TheEventName-father` attribute values and values should be the handler. Otherwise the `data-TheEventName-father` value should be set to true and `delegatedFatherClick` value should be the handler.
+
+For example, you can use the `delegatedFatherClick` binding to associate the element with a specific function.
+
+```html
+<ul data-bind="delegatedHandler: 'click', foreach: items, delegatedFatherClick:removeItem">
+    <li>
+        <span data-bind="text: name"></span>
+        <a href="#" data-click-father="true">x</a>
+    </li>
+</ul>
+``` 
+
+Here is an example how you can use the `delegatedFatherClick` binding to associate multiple elements with diferent functions.
+
+```html
+<ul data-bind="delegatedHandler: 'click', foreach: items, delegatedFatherClick:{remove:removeItem,add:AddItem}">
+    <li>
+        <span data-bind="text: name"></span>
+        <a href="#" data-click-father="removeItem">x</a>
+        <a href="#" data-click-father="AddItem">+/a>
+    </li>
```

Comment:
Assuming the data-click-father attributes key into the object of the delegatedFatherClick binding, should these be

`data-click-father="remove"`   (which would then call the `removeItem` function)
and similar for the add.